### PR TITLE
Update private location image version

### PR DESCRIPTION
**Pull request by MargotLepizzera**
#### What this PR does / why we need it:
This PR updates the version of the private location image used in helm charts.

#### Which issue this PR fixes
*(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*
  - fixes #

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] Documentation has been updated with helm-docs (run: `.github/helm-docs.sh`)
- [x] Chart Version bumped
- [x] `CHANGELOG.md` has beed updated
- [ ] Variables are documented in the `README.md`

### CODE REVIEW

**Review by clamoriniere — COMMENTED** [review]
hello @MargotLepizzera 

you will need to:
* increase the chart version in `chart.yaml`
* run `.github/helm-docs.sh` to update docs in the `README.md`

**Comment by MargotLepizzera** [comment]
@clamoriniere Ran the script locally and there is no changes to be made to the readme.me so I ticked the box - is that sufficient?

File: charts/synthetics-private-location/CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,9 @@
 # Datadog changelog
 
+### 0.5.0
+
+* Update image version to `1.11.0`.
```
Comment (MargotLepizzera) [inline charts/synthetics-private-location/CHANGELOG.md:5]:
```suggestion
* Update private location image version to `1.11.0`.
```

**Review by m-rousse — COMMENTED** [review]
Ok from Synthetics PoV

**Review by clamoriniere — APPROVED** [review]

**Pull request merged**